packages/repository/src/__tests__/acceptance/operation-hooks.acceptance.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {DataSource} from 'loopback-datasource-juggler';
+import {DefaultCrudRepository, juggler} from '../..';
+import {Product, ProductRelations} from '../fixtures/models/product.model';
+
+// This test shows the recommended way how to use @loopback/repository
+// together with existing connectors when building LoopBack applications
+describe('Operation hooks', () => {
+  let repo: ProductRepository;
+  beforeEach(givenProductRepository);
+
+  const beforeSave = 'before save';
+  const afterSave = 'after save';
+  const expectedArray = [beforeSave, afterSave];
+
+  it('supports operation hooks', async () => {
+    const p = await repo.create({slug: 'pencil'});
```

Comment:
code linter complaining about `p` is not being used. 
Should change this to:
```
await repo.create({slug: 'pencil'});
```

Reply:
👍